### DISCUSSION TITLE
How to disable warnHtmlMessage?

### DISCUSSION THREAD

**Discussion by memic84** [nuxt-modules/i18n#1968] [Q&A]
I need to enable HTML in my .json translation files, and according the vue-i18n docs it should be this for the composition api: warnHtmlMessage: false

I've tried adding this to the i18n in nuxt.config and also i18n.vueI18n, but nothing happens.

This is the message i get:

```[plugin:unplugin-vue-i18n] Detected HTML in```

**Comment by trapegen** [comment]
The warnHtmlMessage: false option in vue-i18n disables the warning message that is displayed when HTML tags are found in translation strings. To actually enable HTML in your translations, you need to set the preserveDirectiveContent option to true.

To do this with Nuxt.js and vue-i18n, you can add the preserveDirectiveContent option to the vueI18n object in your nuxt.config.js file

**Reply by memic84** [reply]
Haven't tried that yet, but this worked for me: https://github.com/nuxt-modules/i18n/issues/1970#issuecomment-1491977957
